packages/translation/src/lang/en.json

```diff
@@ -599,7 +599,17 @@
         "label": "Description"
       },
       "url": {
-        "label": "Url"
+        "label": "Url",
+        "placeholder": "https://...."
```

Comment:
I wouldn't put this in translations, because it will look exactly the same for all languages (Same for the ping input)